hw1/02-palindrome.js

```diff
@@ -1,3 +1,24 @@
-const elem = document.querySelector('input');
+// Sources: https://stackoverflow.com/questions/38053729/javascript-how-to-reverse-a-number
+const elem = document.querySelector("input");
 
-elem.addEventListener('input', handleInput);
+elem.addEventListener("input", handleInput);
+
+let item = document.querySelector("#output");
+let out = document.createElement("div");
+item.append(out);
+
+function handleInput(event) {
+  if (elem.value <= 0) {
+    out.setAttribute("class", "text-danger");
+    out.textContent = "Please enter a positive number.";
+  } else {
+    let rev = elem.value.toString().split("").reverse().join("");
```

Comment:
logic of finding palindrome  can be put into a seperate function for easy review